javascript/app.js

```diff
@@ -0,0 +1,254 @@
+	// Are string and reverse equal?
+	if (string.toLowerCase() === stringJoin) {
+		return true;
+	} else {
+		return false;
+	}
+}
+// //  // Also Tried this vvv Can't make it to work...
+// const checkPalindrome = (string) => {
+// 	// Just in case
+// 	console.log('string: ' + string);
+// 	// Declare array
+// 	const stringArray = [];
+// 	// Store string into array .push()
+// 	// Transform string .toLowerCase
+// 	stringArray.push(string.toLowerCase());
+// 	console.log('stringArray: ' + stringArray);
+// 	// Declare stringArrayReverse
+// 	const stringArrayReverse = [];
+// 	// Push into array the following
+// 	// Split word string into array of letter strings .split('')
+// 	// .reverse() them
+// 	// .join('') them back together
+// 	stringArrayReverse.push(string.toLowerCase().split('').reverse().join(''));
+// 	console.log('stringArrayReverse: ' + stringArrayReverse);
+
+// 	while (stringArray !== stringArrayReverse || stringArray == null || stringArrayReverse == null) {
+// 		return false;
+// 	};
+// 	return true
+// }
+
+console.log(checkPalindrome("Radar"));
+//  => true
+console.log(checkPalindrome("Borscht"));
+// => false
+
+// 🔴 **Commit your work.** 
+// The commit message should read: 
+// "Commit 2 - Palindrome".
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+
+
+// 3. Digit Sum
+// Write a function sumDigits that accepts a number and returns the sum of its digits.
+const sumDigits = (number) => {
+	// This is absolutely brilliant (I did NOT come up with this) I'm looking up the math behind it.
+	// Okay so: Modulo 9 counts the number of 10s in a number (and gives us a remainder of 1 each time)
+	return (number - 1) % 9 + 1;
+}
+console.log(sumDigits(42));
+// => 6;
+
+// 🔴 **Commit your work.** 
+// The commit message should read: 
+// "Commit 3 - Digit Sum".
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+
+
+// 4. Pythagoras
+// Write a function calculateSide that takes two arguments: sideA and sideB, and returns the solution for sideC using the Pythagorean theorem.
+const calculateSide = (sideA, sideB) => {
+	return Math.sqrt(Math.pow(sideA, 2) + Math.pow(sideB, 2));
+}
+
+console.log(calculateSide(8, 6));
+// => 10
+
+// 🔴 **Commit your work.** 
+// The commit message should read: 
+// "Commit 4 - Pythagoras".
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+
+
+// 5. Sum Array
+// Write a function sumArray that takes an array as an argument. The array should contain numbers. The function should return the sum of the numbers in the array.
+const sumArray = (array) => {
+	// From what I understand .reduce() makes us able to iterate through an array
+	// According to MDN .reduce() "executes a reducer function [...] on each member of the array resulting in a single output value."
+	return array.reduce((a, b) => a + b, 0)
+}
```

Comment:
i'll take it. make sure you understand what is happening with `.reduce()`... otherwise use another strategy